python/pyspark/ml/wrapper.py

```diff
@@ -34,10 +34,12 @@ class JavaWrapper(Params):
 
     __metaclass__ = ABCMeta
 
-    #: The wrapped Java companion object. Subclasses should initialize
-    #: it properly. The param values in the Java object should be
-    #: synced with the Python wrapper in fit/transform/evaluate/copy.
-    _java_obj = None
```

Comment:
@mengxr @jkbradley it seems like this should not be a static member, right?  From what I could tell, it's not even being used since the assignment is always to `self._java_obj`.  So I made a change to make it a member instead.